app/packages/plugin.js

```diff
@@ -163,13 +163,15 @@ class Plugin extends Package {
             .then((results = []) => {
               return results.map((result) => {
                 const icon = result.icon || this.plugin.icon || 'fa-bolt'
-                const isFontAwesome = icon.indexOf('fa-') === 0 && icon.indexOf('.') === -1
-                result.icon = isFontAwesome ? icon : (icon.indexOf(this.path) === 0 ? icon : path.join(this.path, icon))
-                result.previewCss = this.plugin.css
-                result.pluginName = this.url
-                result.blockId = input.id
-                result.next = this.next.bind(this, result)
-                return result
+                const isFontAwesome = (icon.indexOf('fa-') === 0 && icon.indexOf('.') === -1)
+                const isAbsolutePath = (icon.indexOf('/') === 0 || icon.indexOf(this.path) === 0)
```

Comment:
As discussed in https://github.com/tinytacoteam/zazu/issues/181#issuecomment-276377531 , to support Linux, we need to handle the case of absolute icon path.

'this.path' can only handle the plugin path prefix case, not any absolute path in the system.